Assets/HoloToolkit-Examples/Boundary/Scripts/BoundaryTest.cs

```diff
@@ -19,7 +19,7 @@ private void Start()
         
             if (BoundaryManager.Instance.ContainsObject(gameObject.transform.position))
             {
-                Debug.LogFormat("Object is within established boundary. Position: {0}", gameObject.transform.position);
+                Debug.LogFormat("Object " + name + " is within established boundary. Position: {0}", gameObject.transform.position);
```

Comment:
`Debug.LogFormat("Object {0} is within established boundary. Position: {1}", name, gameObject.transform.position);